src/plugins/companionTranslations.ts

```diff
@@ -50,6 +50,13 @@ export default function companionTranslations(globPattern: string) {
       translations[tag] = await loadTranslations(filePath);
     }
 
+    const languageTags = Object.keys(translations);
+    if (languageTags.indexOf(fallbackLocale) === -1) {
```

Comment:
You can skip the `Object.keys` and check for property existence directly.
```suggestion
    if (!Object.hasOwnProperty(languageTags, fallbackLocale)) {
```